content/release-notes/1.17.0.md

```diff
@@ -0,0 +1,44 @@
+---
+date: 2020-07-13T08:00:00-07:00
+linktitle: "1.17.0"
+title: "1.17.0"
+kubernetes: "1.16, 1.17, and 1.18"
+---
+
+{{<features>}}
+* GitOps - Added Github Enterprise support.
+* Snapshots - Added a "Internal Storage (Default)" option to the providers list for KURL embedded clusters. Which will switch back to the default configuration that velero was set up with when the app was installed.
+* Added `when` field support for config groups.
+{{</features>}}
+
+{{<changes>}}
+* Kotsadm - New Support bundles will be shown at the top of the list instead of the bottom.
+* Kotsadm - Showing warnings on the Version History page in the Admin Console when installing an application with invalid YAML.
+* Kotsadm - Added a page title for the 'View files' and 'Config' tabs.
+* Removed the KOTS metadata that was added to all application manifests. Which included the local application and the version sequence injected as annotations to all deployed manifests.
+* Show the airgap upload progress percentage more granularly.
+* Removed the "Re-analyze" button from the troubleshoot support bundle analysis page.
+* Auto deploy the app's first version when installing the app and no preflight checks are available or preflight checks pass.
+* Update to import k8s 1.18 libraries.
```

Comment:
🤷  Support for later versions of k8s is one thing, but I don't care about which libraries are imported. 
```suggestion
```

Reply:
@austinchambers but kots can be used as a library, and i'd rather include developer release notes than not include them. This is important for someone who might be using kots in a way that we don't document specifically.

Reply:
I suppose developer release notes are possibly best placed in the repository, and not kots.io. It confuses our audiences. So I still think it's best removed. 

Also curious about how you use KOTS as a library?

Reply:
It's a go library (actually kotsadm uses it as a go library, not an executable).

Let's reword this to:

* Update kubernetes dependencies to 1.18

Reply:
Rewording accordingly
```suggestion
* Update to import k8s 1.18 libraries.
```

Reply:
I think we broke the github suggested change here.